frontend/src/App.scss

```diff
@@ -15,6 +15,13 @@ a:visited {
   color: $text-visited
 }
 
+/*|||||| Accessibility fix |||||||*/
+[hidden] {
+  display: none !important;
```

Comment:
I don't know why this wasn't already in there

Reply:
Do we need !important